### ISSUE TITLE
Decide where modifiers (focused, active, etc.) should be in CSS variable names

### ISSUE THREAD

**Issue by ellisonbg** [jupyterlab/jupyterlab#2887]
Our currently naming scheme is inconsistent in this respect. Reminder for post 0.27 work.

**Comment by blink1073** [comment]
From what I can tell after #2984 we are consistent in this (at least in terms of using `jp-mod-focused` and `jp-mod-active` (or their `p-` equivalents).

Used these commands to look:

```
grep -irn --exclude-dir=node_modules --exclude="*.js.map" "focus" . | grep css
grep -irn --exclude-dir=node_modules --exclude="*.js.map" "active" . | grep css
```

**Comment by blink1073** [comment]
Oh, CSS variable names, was thinking of CSS class names.  Looking again.

**Comment by blink1073** [comment]
Ah, what we need is a policy on modifiers.  Should it be `--jp-name-attribute-modifier1-modifier2` or `--jp-name-modifier1-modifier2-attribute`.  I think `selected` should be part of the `name` portion, and the modifiers should go afterward, to mimic CSS rules.

**Comment by blink1073** [comment]
Hmm, looking at it context I think putting *all* modifiers first makes the most sense, regardless of whether they have CSS analogues.  e.g.:

```
--jp-notebook-multiselected-color
--jp-input-background-hover-color
--jp-editor-focused-selected-background
--jp-cell-prompt-not-active-font-color
```

**Comment by ellisonbg** [comment]
I like this suggestion and have implemented it in #3403
